### DISCUSSION TITLE
A pretty formatting under Visual Code

### DISCUSSION THREAD

**Discussion by claudiosa** [vlang/v#14201] [General]
Hi everyone

I am using the Visual Code under Linux to edit  V. Although I had installed the package for V
unfortunately, the codes are not automatically indented. The formatting is done almost all manually
Is there any shortcut to do it? The language is recognized very well.

Thanks once

ccs

PS: I could not any thread on this

**Comment by vladimirmyshkovski** [comment]
Yes, I think we need to improve VSCode Plugin.

For example, in the screenshot below, in my opinion `mut` should have the same color as `fn`. Also, in my opinion, `T` in `< >`, should have different color.

![image](https://user-images.githubusercontent.com/12729789/175805495-ecddb0f2-a0d3-4b71-aba7-586e91f597e4.png)
